queue_job/job.py

```diff
@@ -437,12 +467,10 @@ def description(self):
         else:
             return '%s.%s' % (self.model_name, self.func.__name__)
 
-    @property
+    @cached_truthy_property
```

Comment:
Well, since it only caches truthy values, `lazy_property` would not reflect all it does. Perhaps `lazy_truthy_property`? `truthy_memoize`? I'm open to suggestion, even `lazy_property` if you prefer.

BTW, there is an actual [`lazy_property`](https://pypi.python.org/pypi/lazy-property) Python distribution, but it behaves slightly differently regarding the cached value storage, and it stores anything, even non-truthy values. It would be nice to avoid confusion.